spec/models/event_spec.rb

```diff
@@ -16,12 +16,29 @@
     }
   end
 
+  describe ".upcoming_events" do
+    it "returns upcoming event" do
+      event = FactoryGirl.create(:event)
+      expect(Event.upcoming_events).to eq [event]
+    end
+
+    it "doesn't return past event" do
+      event = FactoryGirl.create(:event, starts_at: 1.hour.ago)
+      expect(Event.upcoming_events).to be_empty
+    end
+
+    it "limits count of returned events" do
+      3.times { FactoryGirl.create(:event) }
+      expect(Event.upcoming_events(2).count).to eq 2
+    end
+  end
+
   describe "#post_to_nation_builder" do
     context "creating new event without remote_id" do
       it "sends call to update NationBuilder" do
-        expect(Integration::NationBuilder).to receive(:create_event)
-          .with({ attributes: event_attributes })
-        FactoryGirl.create(:event, starts_at: start, ends_at: start + 1.hour)
+        expect(Integration::NationBuilder).to receive(:create_event).
+            with({ attributes: event_attributes }) { 1 }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.